routes/api/projects.js

```diff
@@ -7,18 +7,20 @@ const Project = require('../../models/Project');
 const validateProject = require('../../validation/projects');
 
 // Get all projects
-// Will need some pagination / limit logic
+// Will prob need some limit / filtering logic
 router.get('/', (req, res) => {
-  Project.find()
-    .then((projects) => res.json(projects))
+  Project.find({}, { title: 1, images: 1 })
```

Comment:
does this limit the title and images returned to 1? Or is it a remnant from testing?

Reply:
It's the syntax for specifying a subset of fields to return. It's called a projection, can read about it here: https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/